roles/generate-jenkins/templates/Jenkinsfile.j2

```
@@ -988,7 +988,20 @@ pipeline {
                 --shm-size=1gb \
                 -v /var/run/docker.sock:/var/run/docker.sock \
                 -e IMAGE=\"${IMAGE}\" \
-                -e DELAY_START=\"${CI_DELAY}\" \
+{% if repo_vars | select('regex', 'CI_SBOM_TIMEOUT') %}
+                -e SBOM_TIMEOUT=\"${CI_SBOM_TIMEOUT}\" \
+{% endif %}
+{% if repo_vars | select('regex', 'CI_WEB_SCREENSHOT_TIMEOUT') %}
+                -e WEB_SCREENSHOT_TIMEOUT=\"${CI_WEB_SCREENSHOT_TIMEOUT}\" \
+{% endif %}
+{% if repo_vars | select('regex', 'CI_SCREENSHOT_DELAY') %}
+                -e SCREENSHOT_DELAY=\"${CI_SCREENSHOT_DELAY}\" \
```

I used SCREENSHOT_DELAY since I thought WEB_SCREENSHOT_DELAY was in use as the timeout value. But since it was never used anywhere, I think we can use the WEB_ prefix so it's consistent with the others. 

I can update the CI pr.

```suggestion
                -e WEB_SCREENSHOT_DELAY=\"${CI_WEB_SCREENSHOT_DELAY}\" \
```

---

updated